### PR TITLE
[to #67] remove unused function and dead code

### PR DESCRIPTION
**Pull request by zz-jason**
Signed-off-by: Jian Zhang <zjsariel@gmail.com>

### What problem does this PR solve?

Issue Number: [to #67]

Problem Description: TiKV BR is aimed at backup and restoring data for TiKV-only clusters. Functions related to TiDB cluster are duplicated with existing tools, which leads to misunderstanding of the tool and increases code complexity.

### What is changed and how does it work?

remove TiDB related backup and restore commands and some dead codes

### Code changes

- Has interface methods change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: br/pkg/backup/schema.go [hunk 1]
```diff
@@ -46,23 +45,6 @@ type Schemas struct {
 	schemas map[string]*schemaInfo
 }
 
-func newBackupSchemas() *Schemas {
-	return &Schemas{
-		schemas: make(map[string]*schemaInfo),
-	}
-}
-
-func (ss *Schemas) addSchema(
```
Comment (pingyu) [inline br/pkg/backup/schema.go:55]: The whole `Schemas` seems to be unused for us.

File: br/pkg/restore/client.go [hunk 2]
```diff
@@ -780,171 +549,6 @@ func (rc *Client) switchTiKVMode(ctx context.Context, mode import_sstpb.SwitchMo
 	return nil
 }
 
-// GoValidateChecksum forks a goroutine to validate checksum after restore.
-// it returns a channel fires a struct{} when all things get done.
-func (rc *Client) GoValidateChecksum(
```
Comment (pingyu) [inline br/pkg/restore/client.go:785]: Restore to raw won't do the checksum now ? Then I think we will have a TODO to add it.

Comment (zz-jason) [reply br/pkg/restore/client.go:785]: yes, I haven't found the checksum process for RawKV. it should be considered in both backup and restore.

Comment (zz-jason) [reply br/pkg/restore/client.go:785]: yes, I haven't found the checksum process for RawKV. it should be considered in both backup and restore.

File: br/pkg/backup/schema.go (hunk 1 above)
Comment (zz-jason) [reply br/pkg/backup/schema.go:55]: yes. I deleted the whole file in the following commit.

File: br/pkg/backup/client.go [hunk 3]
```diff
@@ -216,275 +206,6 @@ func CheckBackupStorageIsLocked(ctx context.Context, s storage.ExternalStorage)
...
-				return errors.Trace(err)
-			}
-			count++
-		}
-	}
-	log.Debug("get completed jobs", zap.Int("jobs", count))
-	return nil
-}
-
-// BackupRanges make a backup of the given key ranges.
```
[the remark this answers is not in the document]
Comment (zz-jason) [reply br/pkg/backup/client.go:448]: added it back.

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by zeminzhou — APPROVED** [review]

**Pull request merged**